components/ContentsSelection.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+
+export const ContentsSelection = () => {
+    const [selected, setSelected] = useState("Visual Identity");
+
+    return (
+        <>
+            <div className="flex flex-row gap-[5vw] justify-center">
+                <div onClick={() => setSelected("Visual Identity")}>
+                    <p className="text-white">Visual Identity</p>
+                </div>
+                <div onClick={() => setSelected("Preperation Comittee")}>
+                    <p className="text-white">SNU DESIGN WEEK 2023<br/>Preperation Comittee</p>
+                </div>
+                <div onClick={() => setSelected("Exhibition Participants")}>
+                    <p className="text-white">Exhibition<br/>Participants</p>
+                </div>
+                <div onClick={() => setSelected("Thanks To")}>
+                    <p className="text-white">Thanks To</p>
+                </div>
```

Comment:
이 부분은 맵 함수를 이용해서 더욱 간결하게 표현할 수 있을 것 같습니다. 예를 들어 컴포넌트의 props로 토글되는 워딩의 배열을 받는다고 치면 다음과 같이 고쳐볼 수 있을 것 같아요

```suggestion
            // received headingList(Ex. ["Visual Identity", "Preperation Comittee", ... ]) from component props
            headingList.map((heading, index) => {
                return (
                    <div onClick={() => setSelected(index)}>
                        <p className="text-white">{heading}</p>
                    </div>
                )
            }
```